homeassistant/components/climate/ecobee.py

```diff
@@ -269,6 +269,7 @@ def device_state_attributes(self):
             "fan": self.fan,
             "mode": self.mode,
             "operation": operation,
+            ATTR_OPERATION_LIST: self._operation_list,
```

Comment:
Yes, operation_list has nothing to do with that. It is just a constant list which tells you what services the thermostat offers.